src/state/sagas/scenario/LaunchScenario/LaunchScenario.js

```diff
@@ -29,10 +29,17 @@ export function* launchScenario(action) {
     });
 
     // Launch scenario if parameters update succeeded
-    yield call(Api.ScenarioRuns.runScenario, organizationId, workspaceId, scenarioId);
+    const res = yield call(Api.ScenarioRuns.runScenario, organizationId, workspaceId, scenarioId);
```

Comment:
**annoying voice**: Could we call this constant `response` to be coherent with other sagas ? :sweat_smile:

Reply:
Done :heavy_check_mark: